database/models/thermo_transport.py

```diff
@@ -35,6 +35,7 @@ class Thermo(models.Model):
     species = models.ForeignKey(Species, on_delete=models.CASCADE)
     prime_id = models.CharField(blank=True, max_length=11)
     preferred_key = models.CharField(blank=True, help_text="i.e. T 11/97, or J 3/65", max_length=20)
+    # ^^ why is this called preferred_key?
```

Comment:
Not sure. This was there before I was around. Looks like there is a mention in the docstring: 
"Preferred Key (in thermo file, group="prime": What does this mean?) (i.e. ATcT /A, RUS 79)"